OpenOversight/app/config.py

```diff
@@ -34,6 +30,15 @@ class BaseConfig(object):
     OO_MAIL_SENDER = 'OpenOversight <OpenOversight@gmail.com>'
     # OO_ADMIN = os.environ.get('OO_ADMIN')
 
+    # AWS Settings
+    AWS_ACCESS_KEY_ID = os.environ.get('AWS_ACCESS_KEY_ID')
+    AWS_SECRET_ACCESS_KEY = os.environ.get('AWS_SECRET_ACCESS_KEY')
+    AWS_DEFAULT_REGION = 'us-west-2'
+    S3_BUCKET_NAME = 'openoversight'
```

Comment:
No, sorry. I meant that `AWS_DEFAULT_REGION` and `S3_BUCKET_NAME` should be set in .env and therefore not hardcoded to values in `config.py`.